blocks/library/quote/index.js

```diff
@@ -20,6 +20,36 @@ registerBlock( 'core/quote', {
 		citation: html( 'footer' )
 	},
 
+	transforms: {
+		from: [
+			{
+				type: 'block',
+				blocks: [ 'core/text' ],
+				transform: ( { content } ) => {
+					return {
+						value: content,
+					};
+				},
+			},
+		],
+		to: [
+			{
+				type: 'block',
+				blocks: [ 'core/text' ],
+				transform: ( { value, citation } ) => {
+					if ( citation ) {
+						return new Error(
+							'Quote citation would be lost on transform.'
```

Comment:
I think this is an acceptable data loss. I think the razor we should hold any transformation against is that it has to be _non-destructive_, which is fuzzy enough that it would okay this transformation, but disallow transforming it into a gallery for example (which would make no sense). In other words, yes you might lose some formatting, but the _data_ is still there.